app/assets/javascripts/premadeModal.js

```diff
@@ -0,0 +1,18 @@
+var onReadyStrategiesModal = function() {
+	$("#myBtn").click(function() {
+		$("#myModal").show();
```

Comment:
Similarly, `#myModal` should be renamed to something like `#premade_strategies`.